server/controllers/profile.js

```diff
@@ -0,0 +1,62 @@
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+exports.profileCreatePost = asyncHandler(async (req, res) => {
+  const { firstName, lastName, description, availability, email, userName } =
```

Comment:
These variables will need to updated once relevant changes have been made to the Profile model.